### PR TITLE
Cap Pell Grant at the cost of attendance

### LINKED ISSUE #4784: Cap Pell Grant at cost of attendance

### PR DESCRIPTION
**Pull request by CalebPena**
fixes #4784

### CODE REVIEW

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+- bump: patch
+  changes:
+    fixed: Cap the Pell Grant amount at the cost of attendence
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:3]:
```suggestion
    fixed: 
       - Cap the Pell Grant amount at the cost of attendance.
```

**Review by PavelMakarchuk — APPROVED** [review]

File: changelog_entry.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+- bump: patch
+  changes:
+    fixed: 
+       - Cap the Pell Grant amount at the cost of attendance.
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]:
```suggestion
      - Cap the Pell Grant amount at the cost of attendance.
```

File: policyengine_us/variables/gov/ed/pell_grant/pell_grant.py [hunk 3]
```diff
@@ -28,6 +28,6 @@ def formula(person, period, parameters):
                 eligibility == eligibility.possible_values.MAXIMUM,
                 eligibility == eligibility.possible_values.MINIMUM,
             ],
-            [0, p.amount.max, amount],
+            [0, min_(coa, p.amount.max), amount],
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/ed/pell_grant/pell_grant.py:31]: could amount ever exceed coa?

capping as the very final step could be safer if so

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Comment by CalebPena** [comment]
@MaxGhenis when will these changes be reflected in the household API?

**Comment by PavelMakarchuk** [comment]
> @MaxGhenis when will these changes be reflected in the household API?

The changes should already be reflected in the API

**Comment by CalebPena** [comment]
I am getting the expected result on app API, but I am still getting the old result on the household API.

**Comment by MaxGhenis** [comment]
Should be up now

**Pull request merged**